.github/workflows/deploy.yml

```diff
@@ -231,22 +260,56 @@ jobs:
            release_name: Release ${{ steps.tag_version.outputs.new_tag }}
            body: ${{ steps.tag_version.outputs.changelog }}
            commitish: ${{ github.event.inputs.sha }}
-            
-       - name: Trigger Deployment to  QA
-         if: github.ref == 'refs/heads/master' && github.event.inputs.environment == 'Development'
+           prerelease: false
+           draft:      true
+
+       - name: Trigger Deployment to QA
+         if: github.event.inputs.stop == 'no'
          uses: benc-uk/workflow-dispatch@v1.1
          with:
            workflow: Deploy to PaaS
            token: ${{ secrets.ACTIONS_API_ACCESS_TOKEN }}
-           inputs: '{"environment": "Test", "sha": "${{ github.event.inputs.sha }}"}'
+           inputs: '{"environment": "Test", "sha": "${{ github.event.inputs.sha }}" , "release_tag": "${{steps.tag_version.outputs.new_tag}}" }'
+  qa:
+    name: 'Post QA Deployment process'
+    runs-on: ubuntu-latest
+    if: github.event.inputs.environment == 'Test'
+    needs: deploy
+    steps:
+       - name: Checkout
+         uses: actions/checkout@v2
+         with:
+             repository: DFE-Digital/get-into-teaching-frontend-tests
 
-       - name: Slack Notification
-         if: failure() 
-         uses: rtCamp/action-slack-notify@master
+       - name: Run the Tests for Adviser
+         run: |
+             docker run -i --shm-size=1g   -e CYPRESS_HTTPAUTH_PASSWORD -e CYPRESS_HTTPAUTH_USERNAME  -v $PWD:/test -w /test dfedigital/dfecypress:latest \
```

Comment:
My reading of this is that the Cypress tests will become a blocker to release?

One thing to be aware of is this will mean we always need to do manual releases between a 'breaking release' and sushant getting the cypress tests passing again. If thats by design than that should be fine.

I wonder if we want the cypress test suite change to automatically kick off the deploy to QA? So that it can than carry on into prod when the test suite gets fixed?

Reply:
I don't know how cypress would know what it is releasing.
The manual release I have put in, will deliver to production regardless. So we can decide if it is the test thats failing and still just push through.

Another option, would be to make the Cypress tests not be a blocker and just raise a Slack Message if they fail, though allow the pipeline to go foraward